docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
-# Updating an Application
+# Quick Updating an Application
 
 The following procedure describes how to update applications from the Applications page. You can also update applications from the form used to [view application information](</docs/portal/applications/viewing-application-information.md>).
```

Comment:
This paragraph shall be changed. Previously, we demoed how to update application from the Applications page, but now the demos are moved to the application information page.

Reply:
Fixed.

Reply:
I do see the change. Can you double check?
![image](https://user-images.githubusercontent.com/43203744/90619767-300c7e00-e244-11ea-900b-d457bad1062d.png)

Reply:
Fixed by replacing the sentences "The following procedure describes how to update applications from the Applications page. You can also update applications from the application information page." with "The following procedure describes how to update applications."